datadog/api/resources.py

```diff
@@ -384,15 +384,15 @@ class ActionAPIResource(object):
     Actionable API Resource
     """
     @classmethod
-    def _trigger_class_action(cls, method, name, id=None, params=None, **body):
+    def _trigger_class_action(cls, method, action_name, id=None, params=None, **body):
```

Comment:
I renamed the `name` parameter to `action_name` because when you splat the `**body` with a `name` in the dict, it causes there to be a conflict because of two `name` parameters.

Reply:
Test is failing for `embed` because of that name change